pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,45 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level, specially useful for
+        MultiIndex.
+
         Return an Index of values for requested level, equal to the length
```

Comment:
This is pretty repetitive with the summary. I think just the bit about MultiIndex is useful.

    The is primarily useful to get an individual level of values from a MultiIndex,
    but is provided on Index as well for compatability.